Build failed w/CoreDNS v1.4.0 tag

Was trying to work out a dockerfile for dns64, but hit this error:

```
github.com/coredns/coredns/vendor/github.com/openzipkin/zipkin-go-opentracing
github.com/coredns/coredns/plugin/whoami
github.com/mholt/caddy/onevent/hook
github.com/mholt/caddy/onevent
github.com/serverwentdown/dns64
# github.com/serverwentdown/dns64
../../serverwentdown/dns64/dns64.go:65:22: r.DNS64.Proxy.Lookup undefined (type proxy.Proxy has no field or method Lookup)
../../serverwentdown/dns64/setup.go:59:12: undefined: proxy.NewLookup
github.com/coredns/coredns/vendor/github.com/DataDog/dd-trace-go/tracer
github.com/coredns/coredns/vendor/github.com/DataDog/dd-trace-go/opentracing
```
How to replicate it:

```
FROM golang:1.12
ENV GO111MODULE=off
ENV TAG=v1.4.0
RUN mkdir -p $GOPATH/src/github.com/coredns \
 && cd $GOPATH/src/github.com/coredns/ \
 && git clone --branch ${TAG} https://github.com/coredns/coredns \
 && go get -d -v github.com/serverwentdown/dns64 \
 && cd coredns \
 && cp plugin.cfg /tmp \
 && echo dns64:github.com/serverwentdown/dns64 > plugin.cfg \
 && cat /tmp/plugin.cfg >> plugin.cfg \
 && go generate \
 && make \
 && coredns -plugins
```

--- a/README.md
+++ b/README.md
@@ -56,12 +56,16 @@ file example.com.db
 
 ## Installation
 
-```
+```console
 $ go get github.com/coredns/coredns
 $ go get github.com/serverwentdown/dns64
 $ cd $GOPATH/src/github.com/coredns/coredns
 $ vim plugin.cfg
-# Add the line dns64:github.com/serverwentdown/dns64 before the hosts middleware
+
+Add to end of file:
+proxy:github.com/coredns/proxy
+dns64:github.com/serverwentdown/dns64
+
 $ go generate
 $ go build
 $ ./coredns -plugins | grep dns64

--- a/dns64.go
+++ b/dns64.go
@@ -2,24 +2,24 @@
 package dns64
 
 import (
+	"context"
 	"errors"
 	"net"
 	"time"
 
 	"github.com/coredns/coredns/plugin"
 	"github.com/coredns/coredns/plugin/pkg/response"
-	"github.com/coredns/coredns/plugin/proxy"
 	"github.com/coredns/coredns/request"
+	"github.com/coredns/proxy"
 
 	"github.com/miekg/dns"
-	"context"
 )
 
 // DNS64 performs DNS64.
 type DNS64 struct {
-	Next   plugin.Handler
-	Proxy  proxy.Proxy
-	Prefix *net.IPNet
+	Next         plugin.Handler
+	Proxy        proxy.Proxy
+	Prefix       *net.IPNet
 	translateAll bool
 }
 
@@ -56,7 +56,7 @@ func (r *ResponseWriter) WriteMsg(res *dns.Msg) error {
 	// do not modify if there are AAAA records or NameError. continue if NoData or any other error.
 	ty, _ := response.Typify(res, time.Now().UTC())
 	if ty == response.NoError || ty == response.NameError {
-		if hasAAAA(res) && ! r.translateAll {
+		if hasAAAA(res) && !r.translateAll {
 			return r.ResponseWriter.WriteMsg(res)
 		}
 	}

--- a/setup.go
+++ b/setup.go
@@ -3,11 +3,12 @@ package dns64
 import (
 	"net"
 	"strconv"
+
 	"github.com/coredns/coredns/core/dnsserver"
 	"github.com/coredns/coredns/plugin"
-	"github.com/coredns/coredns/plugin/pkg/parse"
 	clog "github.com/coredns/coredns/plugin/pkg/log"
-	"github.com/coredns/coredns/plugin/proxy"
+	"github.com/coredns/coredns/plugin/pkg/parse"
+	"github.com/coredns/proxy"
 
 	"github.com/mholt/caddy"
 )

